neqo-transport/src/recovery.rs

```diff
@@ -288,7 +293,8 @@ impl LossRecovery {
     ) {
         qdebug!([self], "packet {:?}-{} sent.", pn_space, packet_number);
         if sent_packet.ack_eliciting {
-            self.time_of_last_sent_ack_eliciting_packet = Some(sent_packet.time_sent);
+            self.spaces[pn_space].time_of_last_sent_ack_eliciting_packet =
```

Comment:
Rather than accessing the field directly, a setter seems appropriate.  That might help with my suggestion below too.